sense-app/src/main/kotlin/com/exactpro/th2/sense/app/source/processor/SenseProcessor.kt

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2023-2023 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.exactpro.th2.sense.app.source.processor
+
+import com.exactpro.th2.common.grpc.Event as GrpcEvent
+import com.exactpro.th2.common.grpc.EventID
+import com.exactpro.th2.common.grpc.Message
+import com.exactpro.th2.common.util.toInstant
+import com.exactpro.th2.processor.api.IProcessor
+import com.exactpro.th2.sense.api.Event
+import com.exactpro.th2.sense.app.cfg.HttpServerConfiguration
+import com.exactpro.th2.sense.app.notifier.http.SenseHttpServer
+import com.exactpro.th2.sense.app.processor.event.EventProcessorListener
+import com.exactpro.th2.sense.app.source.AbstractSource
+import com.exactpro.th2.sense.app.source.Source
+import com.exactpro.th2.sense.app.statistic.impl.NotificationEventStatistic
+import com.google.auto.service.AutoService
+import com.google.protobuf.TextFormat.shortDebugString
+import io.grpc.Server
+import mu.KotlinLogging
+import java.time.Instant
+import java.util.*
+import java.util.concurrent.ConcurrentLinkedDeque
+
+@AutoService(IProcessor::class)
```

Comment:
I don't think we need this for the processor. Only the factory will be loaded by service loader